packages/react-components/src/lib/ol-map/map.tsx

```diff
@@ -79,9 +84,9 @@ export const Map: React.FC<MapProps> = (props) => {
             format(
               coord as Coordinate,
               '{y}°N {x}°E',
-              COORDINATES_FRACTION_DIFITS
+              COORDINATES_FRACTION_DIGITS
             ),
-          projection: PROJECTION,
+          projection: Proj.WSG84,
```

Comment:
Probably should depends on MAP projection, advice with @syncush

Reply:
If we want to support multiple projections here we need more work on the format. also it shouldn't be based on the map as usually you want WGS84.

Reply:
We should have a format for each projection, the `MousePosition` and `Map` projection should be independent.
It is the programmer's responsibility to match the implementation to the needs.

Reply:
@syncush Actually I'm looking for unification. "Consumer" gives me a PROJECTION_TYPE and we correspondingly use proper mouse tracking presentation. Tracking ability is internals, from my point of view

Reply:
@syncush @CptSchnitz think we should transfer **center** and  **zoom** as well

Reply:
The component is made for us and for 3rd party developers (A.K.A Consumers) and as developers, we should be flexible as much as possible but our application should be strict. 

@CptSchnitz  what do you think?

Reply:
I do think we need to provide the option to set the center an zoom from outside, to be as flexible as possible, but i do think in that case we should separate the view into its own component.
If we do want it we should open a new issue as its out of scope of this one.

Reply:
please create proper issues and link this PR and Issue to them.

Reply:
as I already wrote to @CptSchnitz: "mostly at this time I want to make things easy and unified for us(not on 3rd party)".
As much as we open our components for outside configuration as much we'll see duplicated code in APPs.
Let's think about common needs and incapsulate them inside our components (less freedom)

Reply:
@alebinson for example `VectorTilesLayer` doesn't support translating from WGS84 to Mercator and the other way around, so we have to pass it as an argument.